demos/jobs/jobs_demo_mosquitto/demo_config.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * AWS IoT Device SDK for Embedded C
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef DEMO_CONFIG_H
+#define DEMO_CONFIG_H
+
+/**
+ * @brief The client must send a control packet at least
+ * this often in seconds, or the broker will close the connection.
+ */
+#define MQTT_KEEP_ALIVE         ( 120U )
+
+/**
+ * @brief require acknowledgements of MQTT publish operations
```

Comment:
Can you please update the doxygen comments to be readable sentences for user to understand?
```suggestion
 * @brief The acknowledgements required for MQTT publish operations in the demo.
```

Reply:
The comments are in dictionary definition style.  I don't think anyone who's used a dictionary would have trouble with this.